dax/queryer/translator.go

```diff
@@ -26,15 +27,28 @@ func NewServerlessTranslator(controller dax.Controller) *serverlessTranslator {
 	}
 }
 
+var fbClientCache = map[dax.Address]*featurebase_client.Client{}
```

Comment:
the only thing that worries me about this is whether the lack of cache invalidation will bite us. If a new compute node registers with the controller later, with the same Address as a compute node that has died and been removed on the controller, would this client now be referencing the old, dead compute node?

Reply:
Isn't the Address just the URL?

A client that continues making requests to the URL should work, because if the new node came in with the same address, that URL now goes to the new node. I think.

Reply:
Yeah, it's just the URL. I just wasn't sure if `Client` held on to any information which may no longer apply if the underlying host changed. But you're probably right. This isn't a problem if the client's only state is the URL to which it connects.